nautobot/utilities/tests/test_filters.py

```diff
@@ -991,6 +992,48 @@ def test_device_comments_multiple_value_charfield_iregex_negation(self):
         self.assertEqual(self.DeviceFilterSetWithComments(params, self.device_queryset).qs.count(), 0)
 
 
+class GetFiltersetTestValuesTest(FilterTestCases.BaseFilterTestCase):
+    """Tests for `BaseFilterTestCase.get_filterset_test_values()`."""
+
+    queryset = Site.objects.all()
+    exception_message = "Cannot find valid test data for Site field description"
+
+    @classmethod
+    def setUpTestData(cls):
+        statuses = Status.objects.get_for_model(Site)
+        cls.status_active = statuses.get(slug="active")
+
+        Site.objects.all().delete()
+
+    def test_empty_queryset(self):
+        with self.assertRaisesMessage(ValueError, self.exception_message):
```

Comment:
I didn't know about `assertRaisesMessage`... there are quite a few tests I've written where this would be useful!